qvm/sys-gui-vlc-vm.sls

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# vim: set syntax=yaml ts=2 sw=2 sts=2 et :
+
+##
+# qvm.sys-gui-vlc-vm
+# ==================
+##
+
+/home/user/.config/autostart/qvm-start-daemon.desktop:
+  file.managed:
+    - user: user
+    - mode: 640
+    - makedirs: True
+    - contents: |
+        [Desktop Entry]
+        Name=Qubes Guid/Pacat
+        Comment=Starts GUI/AUDIO daemon for Qubes VMs
+        Icon=qubes
+        Exec=qvm-start-daemon --all --watch
+        Terminal=false
+        Type=Application
```

Comment:
This I believe is not needed anymore, since QubesOS/qubes-core-admin-client#171
The default autostart entry included in the qubes-core-admin-client package should be enough.